packages/protocol/test/identity/federatedattestations.ts

```diff
@@ -886,42 +812,21 @@ contract('FederatedAttestations', (accounts: string[]) => {
     it('should succeed when >1 attestations are registered for (identifier, issuer)', async () => {
       const account2 = accounts[3]
       await signAndRegisterAttestation(identifier1, issuer1, account2, nowUnixTime, signer1)
-      const deleteAttestation = await federatedAttestations.deleteAttestation(
-        identifier1,
-        issuer1,
-        account2,
-        {
-          from: account2,
-        }
-      )
-      assertLogMatches2(deleteAttestation.logs[0], {
-        event: 'AttestationDeleted',
-        args: {
-          identifier: identifier1,
-          issuer: issuer1,
-          account: account2,
-        },
+      await checkStorageState(identifier1, issuer1, 1, account2, nowUnixTime, signer1, 0)
```

Comment:
hmm this `checkStorageState` call after registration in the deletion tests seems a bit redundant; I think since you're already checking this in the registration tests, you can assume that register works properly in your delete tests. (If registration breaks, the registration unit tests should break anyways). I'd say you can delete this check pre-deletion in most cases and then just check after deletion.